algorand-sdk.cabal

```diff
@@ -109,7 +109,7 @@ library
     , msgpack-binary >=0.0.14 && <0.1
     , msgpack-types >=0.0.4 && <0.1
     , safe-exceptions ==0.1.*
-    , scientific >=0.3.3 && <0.3.7
+    , scientific >=0.3.3 && <0.3.8
```

Comment:
According to the [PVP](https://pvp.haskell.org/), two versions with the same `A.B` components can be assumed to be backwards compatible. So we should relax the upper bound to be `< 0.4` (like what was done for the other deps)

Reply:
@gromakovsky made similar change here #21

Reply:
Okay, let's just make sure your change doesn't override his when merging.